packages/playground/src/views/settings.vue

```diff
@@ -0,0 +1,245 @@
+              <template #activator="{ props: tooltipProps }">
+                <div v-bind="tooltipProps">
+                  <VTextField
+                    label="Current Password"
+                    v-model="currentPassword"
+                    v-bind="{ ...passwordInputProps, ...validationProps }"
+                    autocomplete="off"
+                    class="pa-3"
+                  />
+                </div>
+              </template>
+            </v-tooltip>
+          </InputValidator>
+        </PasswordInputWrapper>
+
+        <PasswordInputWrapper #="{ props: passwordInputProps }">
+          <InputValidator
+            :value="newPassword"
+            :rules="[
+              validators.required('Password is required.'),
+              validators.minLength('Password must be at least 6 characters.', 6),
+              validateNewPassword,
+            ]"
+            #="{ props: validationProps }"
+            ref="newPasswordInput"
+          >
+            <v-tooltip location="top right" text="Please enter your current password.">
+              <template #activator="{ props: tooltipProps }">
+                <div v-bind="tooltipProps">
+                  <VTextField
+                    label="New Password"
+                    v-model="newPassword"
+                    v-bind="{ ...passwordInputProps, ...validationProps }"
+                    autocomplete="off"
+                    class="pa-3"
+                  />
+                </div>
+              </template>
+            </v-tooltip>
+          </InputValidator>
+        </PasswordInputWrapper>
+        <PasswordInputWrapper #="{ props: confirmPasswordInputProps }">
+          <InputValidator
+            :value="confirmPassword"
+            :rules="[validators.required('A confirmation password is required.'), validateConfirmPassword]"
+            #="{ props: validationProps }"
+            ref="confirmPasswordInput"
+          >
+            <VTextField
+              label="Confirm Password"
+              v-model="confirmPassword"
+              v-bind="{
+                ...confirmPasswordInputProps,
+                ...validationProps,
+              }"
+              autocomplete="off"
+              class="pa-3"
+            />
+          </InputValidator>
+        </PasswordInputWrapper>
+      </form-validator>
+      <v-card-actions class="justify-end">
+        <v-btn :disabled="!isValidPassword" @click="UpdatePassword" class="justify-end ml-auto"
+          >Update</v-btn
+        ></v-card-actions
+      >
+    </v-card>
+    <v-card class="my-5"
+      ><v-card-title>Timeout</v-card-title>
+
+      <v-tooltip text="Set desired session timeout in seconds">
+        <template v-slot:activator="{ props }">
+          <v-card-text v-bind="props">Adjust Timeout <v-icon icon="mdi-information-outline" /></v-card-text>
+        </template>
+      </v-tooltip>
+      <form-validator v-model="isValidTimeout" valid-on-init>
```

Comment:
Input shows an error although the form was not entered.

![image](https://github.com/user-attachments/assets/9da79b5b-071c-42f8-8945-3c1f6731ba27)

Reply:
yes, because the user should not be able to update timeout unless the timeout entered is different from the current one. And the form is mounted with the current timeout so that user knows what the current value is. what do you think @AhmedHanafy725 ?